### DISCUSSION TITLE
I feel dumb, UVR won't open

### DISCUSSION THREAD

**Discussion by a-punk-by-vampire-weekend** [Anjok07/ultimatevocalremovergui#945] [Q&A]
I'm on Mac OS Mojave (you may laugh now, i have plugins that won't work past mojave so i keep plugging along with it).

I've been using UVR for about 5 months now no problem. Today I upgraded UVR to the new version and it wouldn't open. I cleared it in the security preferences and typed in the terminal commands and still nothing. So I deleted it and copied (same UVR version) from my laptop (same OS, still running fine). It said that it was broken and should be moved to the trash. So then I came back here and redownloaded my older version of UVR. Same result, it didn't even let me enable it in security preferences, and typing in the terminal commands did nothing but trigger a "This application couldn't be opened" message when I tried to force it open. What am I missing? Thanks.

**Comment by xxVOXxx** [comment]
hey which version of UVR installed properly and worked on Mojave?  I'm looking for one and 5.6 isn't opening after install.

**Reply by a-punk-by-vampire-weekend** [reply]
<img width="362" alt="Screen Shot 2024-02-09 at 7 22 30 AM" src="https://github.com/Anjok07/ultimatevocalremovergui/assets/149850718/0a25f4d6-cada-4687-ba70-e93edcd3024f">
This is what I have that functions properly on my laptop. v5.5.1. No idea why it won't work on my iMac.
